webapp-django/crashstats/signature/tests/test_views.py

```diff
@@ -390,6 +390,13 @@ def mocked_supersearch_get(**params):
         assert 'term_counts' in struct
         assert len(struct['term_counts']) == 1
 
+    def test_signature_comments_no_permission(self):
+        """Verify comments are not viewable without view_pii."""
+        url = reverse('signature:signature_comments')
+
+        response = self.client.get(url, {'signature': 'whatever'})
+        assert response.status_code == 403
```

Comment:
This verifies you can't view the comment data if you're not logged in. That's equivalent to logged-in, but no `view_pii` permission, so it covers both cases.